### PR TITLE
My first pull request

### PR DESCRIPTION
**Pull request by mrjuda**
This is Judá's first pull request on Microverse.
The following files were created and edited: index.html + main.css + other config files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (carlos-ssh) [inline index.html:1]: - Kindly add to your `repo` the next missing files: `.stylelintrc.json` and `.hintrc` you can have access from ["Here"](https://github.com/microverseinc/linters-config/tree/master/html-css) or from your `project requirements`.

- After do that kindly initialize your project using `npm init` follow the required steps and then follow the instructions for `Install` and `run` your `Linters` locally. You can have access to this instructions [Here](https://github.com/microverseinc/linters-config/tree/master/html-css).

- Add your `node_modules` to your `.gitignore` once generated.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @mrjuda , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mrjuda) [reply index.html:1]: Done :-)

**Review by Cathella — APPROVED** [review]
## 🟢 PROJECT APPROVED 🟢 🎉 

### Hi @mrjuda 👋🏻, 

You have done a tremendous job implementing all changes requested by reviewers 💪🏻💪🏻! 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**